package.json

```diff
@@ -1,9 +1,14 @@
 {
   "name": "fossbilling",
-  "version": "0.1.0",
+  "version": "0.2.10",
```

Comment:
Can we remove the package ver from here to?
No reason in having it, and it's just another thing that has to be periodically changed